pages/analysis.md

```diff
@@ -1,15 +1,132 @@
+---
 
-If you want to add an image you can do it like this....
+![image]({{ site.baseurl}}/assets/images/Pirates%20and%20actions%20with%20slaves.png)
 
-![Example Image]({{ site.baseurl }}/assets/images/example-image.png)
+The Relative Density comparing all pirated ships and its cargo shows that there
+was not a significant influx to ships with slaves and ships without slaves. As 
+described above, the region was covered with ships that carried slave in around 
+the slave market. The lack of significant to ships with and without slaves shows 
+that pirating slaves were not the main focus of pirates and privateers.
 
-Whether you turn your visualizations into static pictures and put them here or embed them, you MUST discuss your visualizations adequately. That means that whoever is the visualization expert must explain what they think the visualization means. You should explain anything that is not self-apparent from the picture alone. Moreover, you should at least comment on whether you think you can draw broader conclusions from any of the visualizations, either when considered individually or all together.
+---
 
-If your project was less visualization-centric, this is where you at least explain in plain words what you learned through the process of non-visual analysis. For example, if you used text mining not for visualization purposes, but to help you manually find interesting threads of conversation in a body of documents.
+![image]({{ site.baseurl}}/assets/images/Tableau%20Bars%20.png)
 
-You should also make sure to be a cautious scholar, and to think about the limitations of what the visualizations can actually tell you. As indicated by the section name, you are **not** expected to make hard conclusions that upend serious historical debates. Rather, this is a place to explore what might be learned from visual exploration.
+There were nine million-plus slaves, documented, were traded and sold in between
+1699-1827 with eight million making it to the slave marker port. Of the over one 
+million slaves lost, only a small fraction was due to pirates and privateers. In 
+this image, you can see that more slaves lost to sinking ship or unknown reason than
+by ships captured by pirates. 
 
-You must also comment on where you would go next. If these are initial findings, where do you think the best profit would be for any future attempts. This is also where you can talk about pitfalls that limited your ability to learn more about the topic. How might future projects overcome these difficulties?
+Although the data may not be ironclad,  the massive difference in sum demonstrates 
+the pirating of slaves caused no significant effect on the slave market or the 
+countries that participated in the practice.
```

Comment:
Overall: fantastic analysis, you did a great job with the visuals, and above and beyond in using multiple tools.